templates/tests/server-test.yaml

```diff
@@ -46,10 +46,14 @@ spec:
       volumeMounts:
         {{- if .Values.server.volumeMounts }}
```

Comment:
Could also move the if statement one line up, to make it include the key, in example:

```
     {{- if .Values.server.volumeMounts }}
      volumeMounts:
          {{- toYaml .Values.server.volumeMounts | nindent 8}}
     {{- end }}
```

Reply:
Changed it now (better late than never)